PluginDirectories/1/workingon.bundle/options.json

```diff
@@ -0,0 +1,9 @@
+{
+	"options": [
+		{
+			"text": "Your WorkingOn.co Token: ",
+			"type": "text",
+			"key": "token"
+		}
```

Comment:
Is there a URL we can direct people to, where they can get a key? it would make sense to have a link right on the settings panel.